DailyCodingProblem/Program.cs

```diff
@@ -92,5 +97,61 @@ static void Problem2()
                 Console.Write($"{number} ");
             }
         }
+
+        static void Problem3()
+        {
+            Node root = new Node(
+                0,
+                new Node(
+                    1,
+                    new Node(11, null, null),
+                    new Node(12, null, null)),
+                new Node(
+                    2,
+                    new Node(21, null, null),
+                    new Node(22, null, null)));
+
+            Console.WriteLine(SerializeProblem3(root));
+        }
+
+        static string SerializeProblem3(Node root)
+        {
+            if (root == null)
+            {
+                return "null / ";
+            }
+            string serializedTree = string.Empty;
+
+            serializedTree += root.Value.ToString() + " / ";
+
+            serializedTree += SerializeProblem3(root.LeftChild);
+
+            serializedTree += SerializeProblem3(root.RightChild);
+
+            return serializedTree;
+        }
+
+        //static Node DeserializeProblem3(string serializedTree)
+        //{
+
+        //}
+
+        /// <summary>
+        /// Given an array of integers, find the first missing positive integer in linear time and constant space. 
+        /// In other words, find the lowest positive integer that does not exist in the array. The array can contain duplicates and negative numbers as well.
+        /// </summary>
+        static void Problem4()
+        {
+            List<int> numbers = new List<int>() { 3, 4, -1, 1 };
+
+            for (int i = 1; i < int.MaxValue; i++)
+            {
+                if (!numbers.Exists(element => element == i))
```

Comment:
This is a valid way of doing it, but I think the easiest would be to use Contains()